client/apps/edit/components/admin/test/index.coffee

```diff
@@ -99,3 +103,22 @@ describe 'EditAdmin', ->
       @view.$('input[name=credit_url]').val().should.equal 'http://credit'
       @view.$('.edit-email-small-image-url input').val().should.containEql 'img.png'
       @view.$('.edit-email-large-image-url input').val().should.containEql 'img.png'
+
+  describe '#toggleScheduled', ->
+
+    it 'can save the scheduled publish date and time', ->
+      @view.article.set {scheduled_publish_at: null}
+      date = '11/11/11'
+      time = '12:30'
+      dateAndTime = date + ' ' + time
+      global.confirm = -> true
+      @view.$('.edit-admin-input-date').val(date)
+      @view.$('.edit-admin-input-time').val(time)
+      @view.toggleScheduled(@e)
+      @view.article.get('scheduled_publish_at').should.containEql moment(dateAndTime, 'MM/DD/YYYY HH:mm').toDate()
+      delete global.confirm
```

Comment:
Ehh, don't worry about it. It might have larger implications on the other web apps and I'm happy with this implementation—just heads up when modifying globals like this.